meta-iot2000-example/README.md

```diff
@@ -84,12 +84,6 @@ environment.
 This replaces the changes to conf/bblayers.conf documented in
 ()[../meta-iot2000-bsp/README.md]. The changes to conf/local.conf are the same.
```

Comment:
Note: this broken link is fixed [here](https://github.com/siemens/meta-iot2000/pull/23/files#diff-83b9da2db5f8438a48c811daa6692f5fR103).